src/applications/ivc-champva/10-10d-extended/chapters/applicantInformation/sectionOverview.js

```diff
@@ -0,0 +1,15 @@
+import {
+  descriptionUI,
+  titleUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import AddApplicantsDescription from '../../components/FormDescriptions/AddApplicantsDescription';
+import { blankSchema } from '../../definitions';
+import content from '../../locales/en/content.json';
+
+export default {
+  uiSchema: {
```

Comment:
It would be cool if there was something like a `uiOnlySchema` where you didn't have to include a `schema` if it's a `blankSchema`